CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 All notable changes to this project will be documented in this file.
 
+
 ## [4.6.0] - TBD
 
 Wazuh commit: TBD \
 Release report: TBD
+## Added
+
+- Added tests for checking agent status upon ungraceful closure.([#3709](https://github.com/wazuh/wazuh-qa/pull/3709)) \- (Tests)
```

Comment:
```suggestion
- Added tests for checking agent status upon ungraceful closure. ([#4146](https://github.com/wazuh/wazuh-qa/pull/4146)) \- (Tests)
```

Reply:
Changed in https://github.com/wazuh/wazuh-qa/pull/4146/commits/d0592899d807833147849678e691e3da5974918c